### ISSUE TITLE
Make `before` and `after` optional parameters

### ISSUE THREAD

**Issue by blimmer** [yykamei/block-merge-based-on-time#424]
Hello,

First of all, thank you for this awesome GitHub action! It really helped me out 😄 

My use-case is using this GitHub action to enforce code freezes during certain days throughout the US Holidays. I don't have a need to use the `before` or `after` parameters, but they're required. It would be nice if I could only provide `prohibited-days-dates` without providing `before` and `after`.

Luckily, this was pretty easy to work around by setting `before` and `after` to the same value. If I understand the code correctly, this will only block merge daily for one millisecond, which was acceptable for us.

```yaml
jobs:
  block:
    runs-on: ubuntu-latest
    steps:
      - uses: yykamei/block-merge-based-on-time@829adba627e95129329b0a6a7699a1ea3fbd094b
        with:
          # `after` and `before` are both required parameters for this GitHub action. Because we
          # don't block every day based on a certain time, we set these to the same value.
          after: '0:00'
          before: '0:00'
          base-branches: '(default)'
          timezone: America/Denver
          prohibited-days-dates: '2021-11-01/2021-11-02'

```

However, it would be nice if we could instead just use this configuration:

```yaml
jobs:
  block:
    runs-on: ubuntu-latest
    steps:
      - uses: yykamei/block-merge-based-on-time@829adba627e95129329b0a6a7699a1ea3fbd094b
        with:
          base-branches: '(default)'
          timezone: America/Denver
          prohibited-days-dates: '2021-11-01/2021-11-02'
```

**Comment by yykamei** [comment]
Thank you for using my GitHub Action!

It makes sense to make `before` and `after` optional for your use case 👍 
I want to consider making them non-required parameters.

Thanks 😄

**Comment by yykamei** [comment]
@blimmer Thank you for suggesting such a great idea! I changed this action via #425 and have released [v1.2.0](https://github.com/yykamei/block-merge-based-on-time/releases/tag/v1.2.0). Please try the new version!

Thanks 😄
